Probesys/bileto
# imp: Allow emails not to be deleted after collection

## Related issue(s)



https://github.com/Probesys/bileto/issues/525

## How to test manually



- create a mailbox (or edit it)
- choose "mark as read" for the action to take on emails
- send an email to the mailbox
- collect the emails → check a ticket is created and that the email is marked as read (and not deleted)

## Checklist
